src/TextInputWidget.cc

```diff
@@ -229,6 +280,9 @@ TextInputWidget::TextInputWidget(QWidget *parent)
         connect(sendFileBtn_, SIGNAL(clicked()), this, SLOT(openFileSelection()));
         connect(input_, &FilteredTextEdit::message, this, &TextInputWidget::sendTextMessage);
         connect(input_, &FilteredTextEdit::command, this, &TextInputWidget::command);
+        connect(input_, &FilteredTextEdit::image, this, [=](QString name) {
+                emit uploadImage(name);
+        });
```

Comment:
I believe this can be written `connect(input_, &FilteredTextEdit::image, this, uploadImage)`